trustvector.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package ar4si
+
+// TODO(tho) description
```

Comment:
Please add description if you can!

Reply:
will do in https://github.com/veraison/ar4si/issues/5